style.css

```diff
@@ -1,503 +1,1087 @@
+    color: #6070ff;
+    background: #ebebff;
+    border-radius: 1px;
+    display: inline;
+  }
+
+  #bttn {
+    font-family: 'Poppins', Arial, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #396df2;
+    padding: 12px;
+    background: #fff;
+    border: 1px solid #396df2;
+    border-radius: 12px;
+    margin: 15px 0;
+  }
+
+  #bttn:active {
+    background: #2e3de9;
+    color: #fff;
+  }
+
+  #bttn:hover {
+    background: #525ec4;
+    color: #fff;
+  }
+
+  .snap {
+    width: 495px;
+    height: 500px;
+    background: url('images/Snapshoot_Portfolio.png');
+    background-repeat: no-repeat;
+    border-radius: 8px;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .canopy {
+    width: 55px;
+    height: 16px;
+    font-family: 'Poppins', Arial, sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    color: #344563;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    list-style: none;
+    justify-content: space-between;
+  }
+
+  .counter {
+    width: 8px;
+    height: 8px;
+    background: #c1c7d0;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+    justify-content: space-between;
+  }
+
+  .bac {
+    width: 88px;
+    height: 16px;
+    font-family: 'Poppins', Arial, sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    color: #7a869a;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    justify-content: space-between;
+  }
+
+  #bttn-blue {
+    font-family: 'Poppins', Arial, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #fff;
+    padding: 12px;
+    background: #396df2;
+    border: 1px solid #396df2;
+    border-radius: 12px;
+    margin: 15px 0;
+  }
+
+  .section6 {
+    height: 900px;
+    background-color: #fff;
+    border: 1px solid rgb(241, 234, 234);
+    border-radius: 10px;
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    justify-content: space-evenly;
+    padding: 16px;
+    font-family: 'Poppins', Arial, sans-serif;
+    position: relative;
+    z-index: -1;
+    bottom: -170px;
+  }
+
+  .section-desktop {
+    display: block;
+  }
+
+  .social {
+    justify-content: space-between;
+    display: flex;
+    width: 50%;
+  }
+
+  .la {
+    font-family: 'Poppins', Arial, sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 20px;
+    line-height: 24px;
+    align-items: center;
+    color: #000;
+    height: 24px;
+  }
+
+  .fr {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 303px;
+    margin-bottom: 50px;
+  }
+
+  .about-list {
+    display: flex;
+    flex-direction: row;
+  }
```

Comment:
- [ ] The about section also looks different from the original design it has the same differences that I mentioned above for the cards, borders, margins, alignment, and distances. Please 🙏🏽 fix it to look similar to the Figma design 👇🏽 

| Figma design | Original design | 
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/215131827-1fb9289b-81d4-42da-9a31-9cd751bb2a97.png) | ![image](https://user-images.githubusercontent.com/58642949/215131947-81c598ca-e902-4fe4-b57e-c8a6076e9b24.png) |